contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
 ---
-date: 2022-05-13
 title: "In-depth: PostHog vs Amplitude"
-rootPage: /blog
-sidebar: Blog
-showTitle: true
-hideAnchor: true
+date: 2024-01-08
+author: ["andy-vandervell"]
+rootpage: /blog
 featuredImage: ../images/blog/posthog-vs-amplitude/posthog-vs-amplitude.jpg
 featuredImageType: full
-author:
-  - andy-vandervell
-  - joe-martin
 category: General
 tags:
   - Comparisons
 ---
 
 import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
 import { ComparisonRow } from 'components/ComparisonTable/row'
 
-**Contents:**
+Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
 
-- [How is PostHog different to Amplitude?](#how-is-posthog-different-from-amplitude)
-- [PostHog vs Amplitude feature comparison](#feature-comparison)
-- [Integrations and data sources](#integrations-and-data-sources)
-- [Strengths of PostHog](#strengths-of-posthog)
-- [Strengths of Amplitude](#strengths-of-amplitude)
+1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
 
-## How is PostHog different from Amplitude?
+2. **PostHog** is a comprehensive, all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
 
-Over 10,000 companies already use PostHog, and many have switched from Amplitude. Here are a few reasons why.
+## How is PostHog different?
 
-### 1. It's an all-in-one platform
+### 1. We're everything you need in one
 
-PostHog is more than a product analytics tool, it's an all-in-one Product OS that replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flagging, Hotjar for session recording, and [Optimizely](/blog/posthog-vs-optimizely) for A/B testing, or you could just use PostHog for everything. One platform, one price: seamless integration.
+PostHog replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags, Hotjar for session recording, [Statsig](/blog/posthog-vs-statsig) for A/B testing, or you could just use PostHog for everything. One platform, one bill: seamless integration.
```

Comment:
I realize you are doing this for the link, but I don't think anyone would run LaunchDarkly and Statsig in the same stack.